flow/project.py

```diff
@@ -594,10 +624,15 @@ class FlowGroupEntry(object):
         commands to execute.
     :type options:
         str
+    :param aggregate:
+        Aggregator associated with the :py:class:`FlowGroup` to be created.
+    :type aggregate:
+        :py:class`_MakeAggregate`
```

Comment:
If this is only changing the internals, then this should not be exposed to the user yet.